src/main/java/app/entities/CupcakePart.java

```diff
@@ -0,0 +1,46 @@
+package app.entities;
+
+public class CupcakePart {
+    enum Type {
```

Comment:
Nested enum must be public so it can be accessed by other classes via something like `CupcakePart.Type`